src/Layer/include/Reader/Loader.h

```diff
@@ -69,47 +70,46 @@ class NlohmannBuilder
 public:
   static inline glm::mat3 fromMatrix(const nlohmann::json& j)
   {
-    auto v = j.at("matrix").get<std::vector<double>>();
-    assert(v.size() == 6);
+    const auto v = j.value("matrix", std::array<float, 6>{ 1, 0, 0, 1, 0, 0 });
     const auto m = glm::mat3{ glm::vec3{ v[0], v[1], 0 },
                               glm::vec3{ v[2], v[3], 0 },
                               glm::vec3{ v[4], v[5], 1 } };
     return m;
   }
   static inline Bound2 fromBound(const nlohmann::json& j)
   {
-    auto x = j["x"].get<float>(); // convert to skia coordinate system
-    auto y = j["y"].get<float>();
-    auto width = j["width"];
-    auto height = j["height"];
+    auto x = j.value("x", 0.f);
+    auto y = j.value("y", 0.f);
+    auto width = j.value("width", 0.f);
+    auto height = j.value("height", 0.f);
     return Bound2{ x, y, width, height };
   }
 
   static inline std::tuple<Bound2, glm::mat3> fromTransform(const nlohmann::json& j)
   {
-    return { fromBound(j["bounds"]), fromMatrix(j) };
+    return { fromBound(j.value("bounds", nlohmann::json{})), fromMatrix(j) };
   }
 
   static inline void fromObjectCommonProperty(const nlohmann::json& j, PaintNode* obj)
   {
     // all properties that render object cares about
     std::tie(obj->m_bound, obj->m_transform) = fromTransform(j);
-    obj->m_style = j.at("style").get<Style>();
-    obj->m_contextSetting = j.at("contextSettings").get<ContextSetting>();
-    obj->m_maskedBy = j.at("outlineMaskBy").get<std::vector<std::string>>();
-    obj->m_maskType = (EMaskType)j.at("maskType").get<int>();
-    obj->m_overflow = (EOverflow)j.at("overflow").get<int>();
-    obj->m_guid = j.at("id").get<std::string>();
-    obj->setVisible(j.at("visible").get<bool>());
+    obj->setStyle(j.value("style", Style()));
+    obj->setContectSettings(j.value("contextSettings", ContextSetting()));
+    obj->m_maskedBy = j.value("outlineMaskBy", std::vector<std::string>{});
```

Comment:
Why not use setters here? It's not consistent with others. Isn't m_xxx variable private?

Reply:
will be fixed in later PRs